cfgov/jinja2/v1/_includes/molecules/form-field-with-button.html

```diff
@@ -24,43 +24,41 @@
    settings.field.placeholder: The placeholder used for the input.
 
    ========================================================================== #}
+{% if not value %}
+    {% do global_dict.update({
+        'form_field': {
+            'label':       'Form Field with Button',
+            'name':        'm-form-field-with-button',
+            'id':          'form-field-with-button',
+            'placeholder': 'form field + button 4lyf',
+            'type':        'text',
+            'info':        {
+                'source': 'This is a form field with a button. Isn\'t it awesome?'
+            },
+            'required':    false,
+            'btn_text':    'The Awesome Button'
+        }
+    }) %}
+    {% set value = global_dict.form_field %}
+    {% set prototype = true %}
+{% endif %}
 
-{% if value %}
-    <div class="m-form-field-with-button">
-        <div class="form-group">
-            <label for="{{ value.id }}">
-                <b>{{ value.label }}</b>
-                {% if value.required %}
-                    (required)
-                {% endif %}
-            </label>
-            <input id="{{ value.id }}"
-                   type="{{ value.type }}"
-                   placeholder="{{ value.placeholder }}"
-                   class="m-form-field-with-button_field"
-                    {{ "required" if value.required else "" }}>
-        </div>
-        {{ value.info | safe }}
-        <input class="btn btn__full" type="submit" value="{{ value.btn_text }}">
+<div class="m-form-field-with-button">
+    <div class="form-group">
+        <label for="{{ value.id }}">
+            <b>{{ value.label }}</b>
+            {% if value.required %}
```

Comment:
Could also be one line instead of three: `{{ '(required)' if value.required else '' }}`